src/types.mligo

```diff
@@ -313,8 +313,8 @@ type return_with_full_storage = operation list * full_storage
 
 let nil_op = ([] : operation list)
 
-let unfrozen_token_id: nat = 0n
+let unfrozen_token_id: nat = 1n
```

Comment:
I think this should be removed altogether?

Reply:
Same for the `unfrozen_token_id` field in `storage`, which should not longer be necessary.

Reply:
Note that there are probably several connected changes, e.g. the `fa2.mligo` code still checks for `unfrozen_token_id` that are no longer necessary/correct

Reply:
I initially removed it completely. But then we seem to have a lot of tests that required an unfrozen token id. For example to test transfer proposal in `registryDAO`. So added it back with token id of 1.

So if we are to remove it completely, how can we include such tests?

Reply:
@pasqu4le Do you mean that we can consider all tokens except frozen token id as unfrozen implicitly?

Reply:
That's a good question, those tests should be handled depending on the cases.

We might soon have other available `token_id` in the contract because of #54 (and those would not count as "frozen"), but that should be left to be handled by the PR for that issue, right now as of this issue description we should not have an unfrozen token in the contract at all.

So for example: the tests you mentioned for transfer proposals in `registryDAO` should indeed be modified.
Because we no longer have an unfrozen token in the DAO itself, transferring `unfrozen_token_id`s for the DAO in there should result in a failure, however transferring to the governance FA2 token should succeed (I see you already have a dummy contract in there, which I think should be used for these tests as well).

Some other tests (and check in the code) no longer make sense from now on.
So if now we have a check that e.g a transfer can be made only on:
1. `unfrozen_token_id`s and 
2. some cases of `frozen_token_id`s (e.g. by `admin`)

then from now on we should only check for (2) and disallow anything else (in the tests as well), including (1).
Note: if some tests seem useful for the #54 let's only comment them and leave a TODO for it.

Reply:
I think I have covered this. 

One difficulty was with the tests for update operators call for the baseDAO contract. This right now had to be a always failing code, since frozen token id was prohibited as token id for associated with the operator. Since it appeared that we cannot have always failing code in branches, I have changed it to allow adding operator for frozen tokens as long as the operator is the admin.

Apart from this, I had to comment a couple of tests completely like the ones that tests owner and operator FA2 transfers.

Reply:
> I have changed it to allow adding operator for frozen tokens as long as the operator is the admin

Why was that necessary? `admin` was already always able to act as an "operator" for anyone because it already had freedom to transfer `frozen_token`s.